src/main/kotlin/study/calculator/StringCalculator.kt

```diff
@@ -0,0 +1,26 @@
+package study.calculator
+
+object StringCalculator {
+    fun calculate(input: String?): String {
+        return input
```

Comment:
> 메서드 체이닝으로 작성하는 방식이 문제가 될지

오히려 코틀린은 이렇게 체이닝 형태로 작성하는 경우가 더 많기 때문에 문제는 없습니다. 다만 조금 우려되는 부분은 stdlib가 아닌 함수들이 체이닝에 섞여있는데다가 체이닝이 다소 길어서 흐름을 파악하기 위해선 체이닝 메서드 이름만으론 어렵고, 작성된 메서드의 내부를 좀 살펴보아야 할 것 같다는 생각은 드네요.

그리고 중요한 부분은 아니지만 여기선 아래처럼 return을 생략할 순 있겠네요.

```suggestion
    fun calculate(input: String?): String =
        input
```

Reply:
아래 코멘트 주신 부분과 엮어서 수정했습니다.
체이닝 되는 함수 여러 개가 Expression 생성에 관여하는 함수여서 해당 함수들을 이동시키니 체이닝 길이 자체도 짧아져서 가독성이 좋아졌습니다.